app-crypt/p11-kit/p11_kit-0.23.9.recipe

```diff
@@ -0,0 +1,118 @@
+SUMMARY="Provides a way to load and enumerate PKCS#11 modules"
+DESCRIPTION="p11-kit provides a way to load and enumerate PKCS#11 modules. \
+Provides a standard configuration setup for installing PKCS#11 modules \
+in such a way that they're discoverable."
+HOMEPAGE="https://p11-glue.freedesktop.org/p11-kit.html"
+COPYRIGHT="2017 Red Hat, Inc."
+LICENSE="BSD (3-clause)"
+REVISION="1"
+SOURCE_URI="https://github.com/p11-glue/p11-kit/releases/download/$portVersion/p11-kit-$portVersion.tar.gz"
+CHECKSUM_SHA256="e1c1649c335107a8d33cf3762eb7f57b2d0681f0c7d8353627293a58d6b4db63"
+SOURCE_DIR="p11-kit-$portVersion"
+PATCHES="p11_kit-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 ?x86 ?x86_64"
+SECONDARY_ARCHITECTURES="?x86"
+
+libVersion="0.23.9"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
+PROVIDES="
+	p11_kit$secondaryArchSuffix = $portVersion
+	cmd:p11_kit$secondaryArchSuffix = $portVersion
+	cmd:trust$secondaryArchSuffix = $portVersion
+	lib:libp11_kit$secondaryArchSuffix = $libVersionCompat
+	lib:p11_kit_proxy$secondaryArchSuffix = $libVersionCompat
+	"
+
+# TODO: decide if ca_root_certificates is actually necessary
+REQUIRES="
+	haiku$secondaryArchSuffix
+	ca_root_certificates
+	lib:libffi$secondaryArchSuffix
+	lib:libtasn1$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	p11_kit${secondaryArchSuffix}_devel = $portVersion
+	devel:libp11_kit$secondaryArchSuffix = $libVersion
+	"
+REQUIRES_devel="
+	p11_kit$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libffi$secondaryArchSuffix
+	devel:libtasn1$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:awk
+	cmd:find
+	cmd:gcc$secondaryArchSuffix
+	cmd:grep
+	cmd:ld$secondaryArchSuffix
+	cmd:make
+	cmd:makeinfo
+	cmd:pkg_config$secondaryArchSuffix
+	cmd:sed
+	"
+
+USER_SETTINGS_FILES="
+	settings/pkcs11/pkcs11.conf template data/pkcs11/pkcs11.conf.example
+	"
+
+GLOBAL_WRITABLE_FILES="
+	settings/pkcs11/pkcs11.conf keep-old
+	"
+
+defineDebugInfoPackage p11_kit$secondaryArchSuffix \
+	$binDir/p11-kit \
+	$binDir/trust \
+	$libDir/libp11-kit.so.$libVersion \
+	$libDir/p11-kit/p11-kit-remote \
+	$libDir/p11-kit/p11-kit-server \
+	$libDir/pkcs11/p11-kit-client.so \
+	$libDir/pkcs11/p11-kit-trust.so
+
+BUILD()
+{
+	certsInstallDir="$portPackageLinksDir/ca_root_certificates"
+	certsDir="$certsInstallDir/$relativeDataRootDir/ssl"
+	runConfigure ./configure \
+		LDFLAGS="-lnetwork" \
+		--with-trust-paths="${certsDir}/CARootCertificates.pem" \
```

Comment:
I'm still not sure with this one. On my linux, the trust path is at `/etc/ssl/certs/ca-certificates.crt`, which is generated when `update-ca-trust` is run.

So what's the equivalent for `update-ca-trust` on Haiku?

Reply:
On Haiku, [haikuports/haiku-data/ca_root_certificates](https://github.com/haikuports/haikuports/tree/master/haiku-data/ca_root_certificates) does this:
```
INSTALL()
{
	installDir="$dataRootDir/ssl/"
	mkdir -p "$installDir"
	cp ca-bundle-*.crt "$installDir/CARootCertificates.pem"
}
```
which lands in `/boot/system/data/ssl/CARootCertificates.pem`

(But this is readonly.)

Reply:
Thanks. So is this recipe OK then?